fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -39,6 +39,12 @@ def self.bundler?
       return false
     end
 
+    # Do we run from a bundled fastlane, which contains Ruby and OpenSSL?
+    # Usually this means the fastlane directory is ~/.fastlane/bin/
+    def self.contained_fastlane?
+      ENV["FASTLANE_IS_BUNDLED"].to_s.length > 0
```

Comment:
Maybe do an explicit check for `== "true"` here or something, or things will break if someone sets `ENV["FASTLANE_IS_BUNDLED"]="false"` in the future